kirin/core/model.py

```diff
@@ -78,6 +78,7 @@ class TimestampMixin(object):
 
 Db_TripEffect = db.Enum(*[e.name for e in TripEffect], name="trip_effect")
 Db_ModificationType = db.Enum(*[t.name for t in ModificationType], name="modification_type")
+Db_connector_type = db.Enum("cots", "gtfs-rt", name="connector_type", metadata=meta)
```

Comment:
Why different casing than the enums above?
```suggestion
Db_ConnectorType = db.Enum("cots", "gtfs-rt", name="connector_type", metadata=meta)
```